spark3-extensions/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/call.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.sql.catalyst.plans.logical
+
+import org.apache.spark.sql.catalyst.expressions.Expression
+
+case class Call(name: Seq[String], args: Seq[CallArgument]) extends ParsedStatement
+
+case class CallArgument(name: Option[String], expr: Expression)
```

Comment:
Rather than having CallArgument cover both Positional and Named args we could do 

``` scala
sealed trait CallArg
case class NamedArg (name: String, expr: Expression) extends CallArg
case class PositionalArg (expr: Expression) extends CallArg
```

Reply:
Done.